pints/_mcmc/_adaptive_covariance.py

```diff
@@ -39,6 +31,7 @@ def __init__(self, x0, sigma0=None):
         self._current = None
         self._current_log_pdf = None
         self._proposed = None
+        self._r = None
```

Comment:
what is r? variable name more descriptive or an explanatory comment

Reply:
Have changed this.